### PR TITLE
feat: [ANDROAPP-4049] add pie chart for analytics

### PR DESCRIPTION
**Pull request by mmmateos**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 package org.dhis2.commons.featureconfig.model
 
 enum class Feature(val description: String) {
-    ANDROAPP_2557("Local Analytics 2.5")
+    ANDROAPP_2557("Local Analytics 2.5"),
+    ANDROAPP_4049("Pie Chart Analytics")
```
Comment (andresmr) [inline commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt:5]: I think that the idea using ANDROAPP_2557 which is the Epic of the Local Analytics is to gather together all the features of analytics, enabling and disabling it all in a row, so I would use ANDROAPP_2557 as well in you case.

File: dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/DhisAnalyticCharts.kt [hunk 2]
```diff
@@ -11,10 +11,22 @@ class DhisAnalyticCharts @Inject constructor(
...
         return chartsRepository.getAnalyticsForEnrollment(enrollmentUid)
     }
 
+    override fun getProgramVisualizations(programUid: String): List<Graph> {
+        return chartsRepository.getAnalyticsForProgram(programUid)
+    }
+
+    override fun getHomeVisualizations(): List<Graph> {
+        TODO("Not yet implemented")
+    }
```
Comment (andresmr) [inline dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/DhisAnalyticCharts.kt:24]: If we don't need these methods now, why do we declare them?

**Review by andresmr — COMMENTED** [review]
are you updating the SDK in this commit?

**Review by Balcan — APPROVED** [review]

File: dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/DhisAnalyticCharts.kt (hunk 2 above)
Comment (Balcan) [reply dhis_android_analytics/src/main/java/dhis2/org/analytics/charts/DhisAnalyticCharts.kt:24]: Copying the same answer from the radar chart PR:

The idea was to have these charts (Pie and Radar) ready to be used by the visualizations in the search, home and dataSet screens... but we run into a blocker in the SDK (missing objects).
I'm using them now in my next ticket... so If that's ok, i would leave them there.

File: commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt (hunk 1 above)
Comment (mmmateos) [reply commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt:5]: Ah ok, I thought we have to add one feature for each ticket, I'll change it now

File: commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt [hunk 3]
```diff
@@ -2,5 +2,4 @@ package org.dhis2.commons.featureconfig.model
 
 enum class Feature(val description: String) {
     ANDROAPP_2557("Local Analytics 2.5"),
```
Comment (andresmr) [inline commons/src/main/java/org/dhis2/commons/featureconfig/model/Feature.kt:4]: is there a missing comma?

**Review by andresmr — APPROVED** [review]

**Pull request merged**